composer-require-checker-config.json

```diff
@@ -0,0 +1,60 @@
+{
+    "symbol-whitelist" : [
+        "null", "true", "false",
+        "static", "self", "parent",
+        "array", "string", "int", "float", "bool", "iterable", "callable", "void", "object", "mixed", "never",
+        "Psl\\Dict\\associate",
```

Comment:
This is necessary, since `azjezz/psl` uses a custom autoloader, which `maglnet/ComposerRequireChecker` does not understand - see azjezz/psl#310

Reply:
A fix has just been merged, trying it out here to hopefully eliminate this...

Reply:
The release `azjezz/psl:1.9.3` fixes this issue. Worth noting, `azjezz/psl:2.0.x-dev` would work too, but requires min PHP 8.1 but `maglnet/ComposerRequireChecker` chokes on PHP 8.1 syntax currently (thankfully, we don't have any yet, but that is something to be aware of, may cause CI failures later down the line)